src/components/FileExplorer.vue

```diff
@@ -257,6 +328,11 @@ async function initTreeNodes() {
   nodes.value = getTree(props.projectName, localFileInformations.value);
 
   await updateAllFilesStatus();
+
+  // Didn't find a proper way to stub it on shallowMount.
```

Comment:
`// TODO: find a better way to stub it on shallowMount`

Reply:
done